sparkles/tests/test_review.py

```diff
@@ -182,3 +184,113 @@ def test_run_aca_review_function():
         {'text': 'P2: 2.84 less than 3.0 for ER', 'category': 'critical'},
         {'text': 'ER count of 9th (8.9 for -9.9C) mag guide stars 1.91 < 3.0',
          'category': 'critical'}]
+
+
+def test_roll_outside_range():
+    kw = {'att': [-0.82389459, -0.1248412, 0.35722113, 0.42190692],
+          'date': '2019:073:21:55:30.000',
+          'detector': 'ACIS-S',
+          'dither_acq': (7.9992, 7.9992),
+          'dither_guide': (7.9992, 7.9992),
+          'focus_offset': 0.0,
+          'man_angle': 122.97035882921071,
+          'n_acq': 8,
+          'n_fid': 0,
+          'n_guide': 8,
+          'obsid': 48334.0,
+          'sim_offset': 0.0,
+          't_ccd_acq': -10.257559323423214,
+          't_ccd_guide': -10.25810835536192}
+    aca = get_aca_catalog(**kw)
+    acar = aca.get_review_table()
+    acar.get_roll_options()
+    assert Quat(kw['att']).roll <= acar.roll_info['roll_max']
+    assert Quat(kw['att']).roll >= acar.roll_info['roll_min']
+
+
+def test_calc_targ_from_aca():
+    """
+    Confirm _calc_targ_from_aca seems to do the right thing based on obsid
+    This does a bit too much processing for what should be a lightweight test.
+    """
+    obs_kwargs = KWARGS_48464
+    aca_er = get_aca_catalog(**obs_kwargs)
+    acar_er = aca_er.get_review_table()
+    q_targ = acar_er._calc_targ_from_aca(acar_er.att, 0, 0)
+    # Cheat and use the string reprs for comparison to 8 decimals
+    assert Quat(obs_kwargs['att']).__repr__() == q_targ.__repr__()
+
+    or_obs_kwargs = obs_kwargs.copy()
```

Comment:
For testing you can use the back door and make this simpler (and easier to understand later):
```
# Now treat this as an OR by forcing the object to believe it is so,
# and confirm the expected HRMA to ACA aimpoint offset of about 70 arcsec.
acar._is_OR = True
q_targ = acar._calc_targ_from_aca(acar.att, 0, 0)
assert np.isclose(acar.att.dq(q_targ).yaw * 3600, 69.59, atol=0.01, rtol=0)
```